loopchain/blockchain/blockchain.py

```diff
@@ -1072,10 +1072,19 @@ def block_loads(self, block_dumped: bytes) -> 'Block':
         block_dumped = zlib.decompress(block_dumped)
         block_json = block_dumped.decode(encoding=conf.PEER_DATA_ENCODING)
         block_serialized = json.loads(block_json)
-        block_height = self.__block_versioner.get_height(block_serialized)
-        block_version = self.__block_versioner.get_version(block_height)
-        block_serializer = BlockSerializer.new(block_version, self.__tx_versioner)
-        return block_serializer.deserialize(block_serialized)
+        try:
+            block_height = self.__block_versioner.get_height(block_serialized)
+            block_version = self.__block_versioner.get_version(block_height)
+            block_serializer = BlockSerializer.new(block_version, self.__tx_versioner)
+            block = block_serializer.deserialize(block_serialized)
+        except AttributeError:  # no serializer above v1.0
+            from loopchain.blockchain.blocks.v1_0 import Block as Block_V1_0
+            block = Block_V1_0._deserialize(**block_serialized)
```

Comment:
Is there any reason to deserialize block directly, without using `BlockSerializer` ?

Reply:
I have a same question too.

Reply:
I have a same question too. 😅

Reply:
Fixed!